lib/findExports.js

```diff
@@ -37,6 +37,31 @@ function findESNamedExports(node) {
   return result;
 }
 
+function resolveNestedNamedExports(node, absolutePathToFile) {
+  if (node.type === 'ConditionalExpression') {
+    // Potential ternary-style export - we pick the first one
+    // module.exports = foo ? require('a') : require('b');
+    return resolveNestedNamedExports(node.consequent, absolutePathToFile);
+  }
+  if (
+    node.type === 'CallExpression' &&
+    node.callee.name === 'require' &&
+    node.arguments.length === 1 &&
+    node.arguments[0].type === 'StringLiteral'
```

Comment:
unrelated: I wonder if conditionals like this would be clearer if they were wrapped up in well-named functions. Here something like `isNodeRequireStatement`?